pom.xml

```diff
@@ -221,6 +222,7 @@
     <module>java-generator</module>
     <module>httpclient-okhttp</module>
     <module>httpclient-vertx</module>
+    <module>log4j</module>
```

Comment:
To be able to properly generate our BOM this module should also be added as a dependency to the dependencyManagement section.

Reply:
Done